README.md

```diff
@@ -179,6 +179,21 @@ your Firebase project (as described below).
       * Delete references to `generativelanguage.googleapis.com`, including the
         `google_service_usage_consumer_quota_override` resource entry in `main.tf`.
 
+## Troubleshooting
+
+```
+Error when reading or editing Project "<project-id>": Get "https://cloudresourcemanager.googleapis.com/v1/projects/<project-id>?alt=json&prettyPrint=false":
+compute: Received 500
+```
+
+This is because authentication timed-out. To resolve,
+
+1. Open a new terminal
+1. Run `terraform apply --auto-approve` in the terminal, click continue when prompted again to login
+1. Wait till the command completes
+1. Reload the web preview if necessary
+
+
```

Comment:
```suggestion
This error happens because authentication has timed-out. Here's how to resolve it:

1.  In Project IDX, open a new terminal.
1.  In the terminal, run `terraform apply --auto-approve`, and then click continue when prompted to log in.
1.  Wait until the command completes.
1.  Reload the web preview, if necessary.

```